app/services/articles/builder.rb

```diff
@@ -91,7 +91,7 @@ def user_editor_v2
 
     def user_editor_v1
       Article.new(
-        body_markdown: "---\ntitle: \npublished: false\ndescription: \ntags: \n---\n\n",
+        body_markdown: "---\ntitle: \npublished: false\ndescription: \ntags: \n//cover_image: direct_url_to_image.jpg\n---\n\n",
```

Comment:
I would change this to `// cover_image: https://direct_url_to_image.jpg` so it's clearer :D

Reply:
done.